R/default-scoring-rules.R

```diff
@@ -0,0 +1,155 @@
+#' @title Select Scoring Rules From A List of Possibilities
+#' @description Helper function to return only the scoring rules selected by
+#' the user from a list of possible scoring rules.
+#' @param possibilities A list of scoring rules.
+#' @param select A character vector of scoring rules to select from the list.
+#' If `select = all` (the default), all possible scoring rules are returned.
```

Comment:
It looks like the default is being missing (and then this is handled as all).

Reply:
Good catch, I updated the default.